### ISSUE TITLE
custom distribution.yaml file to be included locally

### ISSUE THREAD

**Issue by xinity** [devops-works/binenv#253]
hey there,

it's an awesome project i've discovered here, hoping it will leave for a long time onward :)

i've been wondering if there is a possibility to create our own distribution.yaml file and have it "included" in the default one ?

something like:
```
if custom-distribution.yaml exist
then
include custom-distribution.yaml 
```

that would be so freaking cool :)

keep up the awesome work anyway !

**Comment by xinity** [comment]
Kowabonga!!!!!

**Comment by leucos** [comment]
released in [v0.20.0](https://github.com/devops-works/binenv/releases/tag/v0.20.0) :tada:

**Comment by leucos** [comment]
FTR new binenv release lets you use a token to fetch releases from private repos (GitHub only for now)

See https://github.com/devops-works/binenv?tab=readme-ov-file#using-custom-distributions-file-and-private-gitlab-repos for more info.
